e2e/testcases/status_enablement_test.go

```diff
@@ -46,13 +46,13 @@ func TestStatusEnabledAndDisabled(t *testing.T) {
 	rootSync := k8sobjects.RootSyncObjectV1Alpha1(configsync.RootSyncName)
 	// Override the statusMode for root-reconciler
 	nt.MustMergePatch(rootSync, `{"spec": {"override": {"statusMode": "disabled"}}}`)
-	nt.Must(nt.WatchForAllSyncs(nomostest.SkipAllResourceGroupChecks()))
```

Comment:
oops. this was supposed to go in https://github.com/GoogleContainerTools/kpt-config-sync/pull/1600